drivers/hmis/app/models/hmis/filter/household_filter.rb

```diff
@@ -19,11 +19,13 @@ def filter_scope(scope)
   def with_statuses(scope)
     with_filter(scope, :status) do
       if input.status.present?
-        ids = []
+        return scope.active if input.status == ['ACTIVE'] || input.status.sort == ['ACTIVE', 'INCOMPLETE'].sort
+        return scope if input.status.sort == ['ACTIVE', 'INCOMPLETE', 'EXITED'].sort
 
-        ids += scope.merge(Hmis::Hud::Enrollment.active).pluck(:id) if input.status.include?('ACTIVE')
-        ids += scope.merge(Hmis::Hud::Enrollment.incomplete).pluck(:id) if input.status.include?('INCOMPLETE')
-        ids += scope.merge(Hmis::Hud::Enrollment.exited).pluck(:id) if input.status.include?('EXITED')
+        ids = []
+        ids += scope.active.pluck(:id) if input.status.include?('ACTIVE')
+        ids += scope.in_progress.pluck(:id) if input.status.include?('INCOMPLETE')
+        ids += scope.exited.pluck(:id) if input.status.include?('EXITED')
```

Comment:
I went ahead and used the Household scopes here too. I think that's the whole idea of using a view, is that we shouldn't necessarily need to join with enrollment